Project/PrivateVault App/Code/UI/Gallery/GalleryView.swift

```diff
@@ -25,14 +25,16 @@ struct GalleryView: View {
 	@State var currentAlert: AlertItem?
 	@State var previewSelection: PreviewSelection?
 	@State var itemBeingDeleted: StoredItem?
+	@State private var folder: Folder?
 	@Binding var isLocked: Bool
 	
 	@FetchRequest(sortDescriptors: [], animation: .default)
 	var tags: FetchedResults<Tag>
 	
 	var body: some View {
 		ZStack {
-			GalleryGridView(filter: filter, multipleSelection: $multipleSelection, selectedItems: $selectedItems, selection: select, contextMenu: contextMenu)
+			// swiftlint:disable line_length
+			GalleryGridView(filter: filter, multipleSelection: $multipleSelection, selectedItems: $selectedItems, folder: $folder, selection: select, contextMenu: contextMenu, folderContextMenu: folderContextMenu)
```

Comment:
This line could be split into multiple lines, with one argument per line, instead of disabling the SwiftLint rule.